Doc/whatsnew/3.10.rst

```diff
@@ -299,6 +299,10 @@ Deprecated
   as appropriate to help identify code which needs updating during
   this transition.
 
+* ``sqlite3.OptimizedUnicode`` has been deprecated. It has been undocumented
+  and obsolete since Python 3.3.
```

Comment:
Can you announce when it will be removed? https://www.python.org/dev/peps/pep-0387/ prevents to remove it before Python 3.12. So I suggest Python 3.12.

Reply:
You may also explain that it is simply an alias to the builtin str type.

Reply:
Of course!

Reply:
How about this? b106c9e